.env_sample

```diff
@@ -15,8 +15,9 @@ AWS_SECRET_ACCESS_KEY=
 AWS_STORAGE_BUCKET_NAME=public
 AWS_STORAGE_PRIVATE_BUCKET_NAME=private
 AWS_S3_CALLING_FORMAT=boto.s3.connection.OrdinaryCallingFormat
-AWS_S3_HOST=s3-us-west-2.amazonaws.com
+AWS_S3_HOST=s3.amazonaws.com
 AWS_QUERYSTRING_AUTH=False
+S3DIRECT_REGION=us-west-2
```

Comment:
Nevermind, I guess it doesn't make sense if it's set to `AWS_S3_HOST=s3.amazonaws.com` because that won't work =\